### PR TITLE
Bratseth/vip logic take 2

### PR DESCRIPTION
**Pull request by bratseth**
@baldersheim don't merge, this is just to inform you that I'm on it

### CODE REVIEW

**Comment by bratseth** [comment]
@baldersheim please review

**Comment by bratseth** [comment]
This test is way too slow, but I think I'm out of time for today.
Also, I think we can gc the code in prelude.cluster now.

File: container-search/src/main/java/com/yahoo/search/cluster/ClusterSearcher.java [hunk 1]
```diff
@@ -82,6 +82,7 @@ public final void ping(T p, Executor executor) {
             pong = new Pong(ErrorMessage.createUnspecifiedError("Ping was interrupted: " + p));
             logThrowable = e;
         } catch (ExecutionException e) {
+            e.printStackTrace();
```
Comment (baldersheim) [inline container-search/src/main/java/com/yahoo/search/cluster/ClusterSearcher.java]: Leftover debug code ?

Comment (bratseth) [reply container-search/src/main/java/com/yahoo/search/cluster/ClusterSearcher.java]: Yes, removed, thanks.

**Comment by bratseth** [comment]
From the PR build it looks like this test is also unstable. I think we need to write it to work in a single thread.

**Comment by baldersheim** [comment]
It is limited by the ping rate, I guess the simplest is to increase that.
I will run it some times and see before I merge.

**Comment by baldersheim** [comment]
I made the test drive the pings itself. Hopefully that should make it stable.

**Review by bratseth — COMMENTED** [review]
👍

**Pull request merged**